### ISSUE TITLE
mineflayer.vec3 is still in the api doc

### ISSUE THREAD

**Issue by plexigras** [PrismarineJS/mineflayer#471]
mineflayer.vec3 got removed [here](https://github.com/PrismarineJS/mineflayer/commit/cdc5be4cde64d31bb3ff11d6cdddd9fdee447be7#diff-168726dbe96b3ce427e7fedce31bb0bcL39) which breaks some plugins and is still in the [api doc](https://github.com/PrismarineJS/mineflayer/blob/master/doc/api.md#vec3)

**Comment by rom1504** [comment]
Yes I removed it.
The doc still mention that the class exists in vec3 yes. But it's not in mineflayer.vec3 as it used to be.

What plugins does it break ? I fixed mineflayer-navigate and blockfinder already.

This is an intended change as part of removing some other classes that are now independent modules (see https://github.com/PrismarineJS/mineflayer/blob/master/doc/history.md#200 and corresponding commits)

This is why the new version is mineflayer 2.0.0 and not 1.x
